docs/source/05_data/01_data_catalog.md

```diff
@@ -589,3 +590,34 @@ io.save("ranked", ranked)
 ```
 
 > *Note:* Saving `None` to a dataset is not allowed!
+
+# Using Minio (S3 API Compatible Storage) in Data Catalog
+Minio has a S3 Compatible API, if you have security concern, you may consider hosting the storage within your network.
+
+The easiest way to setup Minio is to run a docker.
+
+docker run -p 9000:9000 \
+  -e "MINIO_ACCESS_KEY=AKIAIOSFODNN7EXAMPLE" \
+  -e "MINIO_SECRET_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY" \
+  minio/minio server /data
+
+  You can then access Minio with http://localhost:9000, create bucket and add files like it is on S3.
+
+In your credentials.yml, you need to define the access key, token, and the endpoint_url
+```yaml
+dev_minio:
+  key: AKIAIOSFODNN7EXAMPLE # Assuming you use the default configuration
+  secret: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+  cleitn_kwargs:
+    endpoint_url : 'http://localhost:9000' # default for minio
+```
+
+Assume that you already have a bucket named "kedro-test" and a file "test.csv" get upload to the Minio server. You can simply define it in the Data Catalog and use it just like any other dataset.
+```yaml
+test:
+  type: pandas.CSVDataSet
+  filepath: s3://kedro-test/test.csv
+  credentials: dev_minio
+```
+
+Note that although the filepath use `s3`, it does not really mean your file is stored in s3, it simply tells kedro to use as a s3 endpoint. You should define the real URL in the `credentials` instead.
```

Comment:
```suggestion
Note that although the filepath uses `s3`, it does not mean that the file is stored in an S3 bucket. But it is telling Kedro to use S3 as an endpoint. The real URL should be defined in the `credentials.yml` instead.
```
I was a bit confused about the S3 endpoint part and might have bamboozled myself there, @noklam.

Thoughts, @921kiyo?

Reply:
I am confused that the paragraph you are commenting are removed long time ago.  I am not sure why you still see it.

This is a screenshot of the end of the markdown file in my master branch![image](https://user-images.githubusercontent.com/18221871/95979313-063f9400-0e4e-11eb-871a-46f4406f8d78.png)

Reply:
No problem! Let me dig here to find out why.